### PR TITLE
[FEAT] Add complex diagram image on the What sub-section

### LINKED ISSUE #22: [FEAT] Improve the Theme
### Tasks

- [x] logo in the about section: set dimension for desktop and mobile. See #28
- [x] 404 page: set image dimension for desktop and mobile: #40 
- [x] Cards: add border to better highlight them as we are using a white background: #38 
- [x] Put complex diagram image on the Why sub-section #81
- [ ] Put monitoring image on second What sub-section #122
- [x] Change the icons in HelmetContent (for media): try to use the svg image directly. See #126 and  also #58 
- [x] use a smaller image for the "automated test" blog post (currently 1.7MB!). Better with #25.
- [x] logo: manage background color when displayed in the header and footer. See #24

### PR DESCRIPTION
**Pull request by csouchet**
Closes #22

### CODE REVIEW

**Comment by csouchet** [comment]
@tbouffard Can you try to build the project on Linux ?
I have no problem on MacOs 🙁

**Comment by tbouffard** [comment]
about the gatsby error, I reproduce the issue
I notice that on my local machine and on Github Actions, we have

> $node --version
> v14.17.4
> $npm --version
> 6.14.14

The pakage-lock.json model has been upgraded and is now using the npm@7 format: `"lockfileVersion": 2`
When running with npm@6, we have this message and the dependencies resolution may not be accurate
> npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!

**Comment by csouchet** [comment]
> about the gatsby error, I reproduce the issue
> I notice that on my local machine and on Github Actions, we have
> 
> > $node --version
> > v14.17.4
> > $npm --version
> > 6.14.14
> 
> The pakage-lock.json model has been upgraded and is now using the npm@7 format: `"lockfileVersion": 2`
> When running with npm@6, we have this message and the dependencies resolution may not be accurate
> 
> > npm WARN read-shrinkwrap This version of npm is compatible with lockfileVersion@1, but package-lock.json was generated for lockfileVersion@2. I'll try to do my best with it!
> 
> I also see warnings about babel dependencies like the following one
> 
> > @babel/preset-typescript@7.14.5 requires a peer of @babel/core@^7.0.0-0 but none is installed. You must install peer dependencies yourself.

It's weird, because I don't have those message, and my environment:
```
➜  ~ node --version
v14.17.5
➜  ~ npm --version
6.14.14
```

I think it would be better if you regenerate the `package-lock.json` from 0.

**Comment by tbouffard** [comment]
package-lock.json change from lockfileVersion 1 to lockfileVersion 2 introduced by 4c3c95ec

I am still investigating

File: package.json [hunk 1]
```diff
@@ -18,18 +18,20 @@
     "lint-check": "eslint \"*/**/*.{js,ts,tsx}\" --max-warnings 0"
   },
   "dependencies": {
+    "@babel/core": "^7.15.0",
+    "@mdx-js/mdx": "^1.6.22",
+    "@mdx-js/react": "^1.6.22",
```
Comment (tbouffard) [inline package.json:23]: ⚠️ move to bottom to avoid introducing useless changes

File: src/content/AboutContent.tsx [hunk 2]
```diff
@@ -1,37 +1,39 @@
...
-- monitoring of all process instances for a selected process
-- prediction of a process instance execution 
-- other process data 
-- drill down through process execution flow
-- related resources data
-- and more…`,
+    mdx: <Content3 />,
+    /*    image: {
+      position: 'bottom',
+    },*/
```
Comment (tbouffard) [inline src/content/AboutContent.tsx:33]: ⚠️ remove commented code
```suggestion
```

Comment (csouchet) [reply src/content/AboutContent.tsx]: Good catch :)

File: package.json (hunk 1 above)
Comment (tbouffard) [reply package.json]: done in c3df87d

**Review by tbouffard — APPROVED** [review]
Nice work and refactoring

**Pull request merged**